<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="testcases.workflows.FundOrderFlowTests" tests="10" time="6772.776" ignored="9" failures="0" timestamp="03 Apr 2018 02:23:46 GMT" skipped="0" hostname="BANCUSTOMQA05" errors="1">
  <testcase name="CITI_Vestima_Fund_Order" time="552.869" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="CITI_Fund_Order_Entry_CIS" time="967.006" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="CITI_Fund_Order_Entry_CIS" time="914.831" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="CITI_Fund_Order_Entry_CIS" time="885.899" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="CITI_Fund_Order_Entry_CIS" time="876.341" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="CITI_Vestima_Fund_Order" time="553.405" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="CITI_Vestima_Fund_Order" time="552.816" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="end_testsuite_update_results" time="0.000" classname="testcases.workflows.FundOrderFlowTests">
    <error type="java.lang.IllegalArgumentException" message="wrong number of arguments">
      <![CDATA[java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:357)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
    </error>
  </testcase> <!-- end_testsuite_update_results -->
  <testcase name="CITI_Fund_Order_Entry_CIS" time="911.297" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="CITI_Vestima_Fund_Order" time="558.312" classname="testcases.workflows.FundOrderFlowTests"/>
  <testcase name="CITI_Funds_Entitlement_OperationsUser_Sing" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Funds_Entitlement_OperationsUser_Sing -->
  <testcase name="CITI_Funds_Entitlement_ViewAccessLogin" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Funds_Entitlement_ViewAccessLogin -->
  <testcase name="CITI_Fund_REDM_With_Holdings_CIS" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_REDM_With_Holdings_CIS -->
  <testcase name="CITI_Fund_Settled_Trade_FundSettle" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Settled_Trade_FundSettle -->
  <testcase name="CITI_Funds_Entitlement_OperationsUser_Hk" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Funds_Entitlement_OperationsUser_Hk -->
  <testcase name="CITI_Multihub_Routing_FS" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Multihub_Routing_FS -->
  <testcase name="CITI_Funds_Entitlement_ApplicationAccess" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Funds_Entitlement_ApplicationAccess -->
  <testcase name="CITI_Funds_Entitlement_OperationsUser_Dublin" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Funds_Entitlement_OperationsUser_Dublin -->
  <testcase name="CITI_Funds_Entitlement_OperationsUser_Tampa" classname="testcases.workflows.FundOrderFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Funds_Entitlement_OperationsUser_Tampa -->
</testsuite> <!-- testcases.workflows.FundOrderFlowTests -->
